codecs/imagequant/Makefile

```diff
@@ -0,0 +1,33 @@
+CODEC_DIR := node_modules/libimagequant
+CODEC_OUT_RELATIVE := libimagequant.a
+CODEC_OUT := $(addprefix $(CODEC_DIR)/, $(CODEC_OUT_RELATIVE))
+OUT_JS := imagequant.js
+OUT_WASM := $(OUT_JS:.js=.wasm)
+
+.PHONY: all clean
+
+all: $(OUT_JS)
+
+%.js: %.cpp $(CODEC_OUT)
```

Comment:
I did not miss Makefiles lol

Reply:
Think of `&` as usual `*` in wildcards and it gets easier :D